### PR TITLE
`iOS Devices are not yet supported` Displayed only when iOS physical device is selected

### PR DESCRIPTION
**Pull request by lijy91**

### CODE REVIEW

**Comment by lijy91** [comment]
@passy can you review this?

File: src/plugins/network/RequestDetails.js [hunk 1]
```diff
@@ -532,6 +532,7 @@ class JSONTextFormatter {
   format = (body: string, contentType: string) => {
     if (
       contentType.startsWith('application/json') ||
+      contentType.startsWith('application/hal+json') ||
```
Comment (passy) [inline src/plugins/network/RequestDetails.js]: This stuff here seems unrelated. Could you split this up into a separate PR? :)

Comment (lijy91) [reply src/plugins/network/RequestDetails.js]: I split into two pr, please review again
